sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -97,6 +97,13 @@ private[spark] object HiveUtils extends Logging {
     .booleanConf
     .createWithDefault(true)
 
+  val CONVERT_METASTORE_PARQUET_EXCLUDED_TABLES =
+    buildConf("spark.sql.hive.convertMetastoreParquet.excludedTables")
+    .doc("A comma-separated list of Parquet table names, which do not use the built-in Parquet" +
+      "reader and writer when \"spark.sql.hive.convertMetastoreParquet\" is true.")
```

Comment:
Why don't we simply disable `spark.sql.hive.convertMetastoreParquet` in this case? It looks going to make the codes super complicated.

Reply:
Thanks @HyukjinKwon 
Take another example, in a SQL statement, this SQL statement  needs to read many hive Parquet tables, but `TableA` can't use the built-in Parquet reader and writer,  if we disable `spark.sql.hive.convertMetastoreParquet`, the other tables can't use the built-in Parquet reader and writer too,  we know, the performance of  built-in Parquet reader and writer  is much better,  this will affect the performance of the SQL statement.

Reply:
What I am not sure is that if we really need this fine-grained for this optimization. BTW, why `TableA` wasn't able to use built-in Parquet reader and writer in your case?

Reply:
Data is incompatible, but our tests are not in the latest version.
I think that since we have this configuration `spark. sql. hive. convertMetastoreParquet`, the reason is that it may have incompatible problems.

Reply:
I think we should fix either the data or this optimization compatible. Spark started to have a huge bunch of configurations, and I think we should better avoid it.

Reply:
Ok, I will this PR, thanks